### PR TITLE
python3Packages.gpapi: unbreak

### PR DESCRIPTION
**Pull request by DavHau**
ZHF: #122042

Upstream project seems unmaintained since Nov 2019. Should this package better be removed?

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122334` at e4297f23 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.gpapi</li>
    <li>python38Packages.gplaycli</li>
    <li>python39Packages.gpapi</li>
    <li>python39Packages.gplaycli</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/gpapi/default.nix:25:5:
```
   |
25 |     license = licenses.gpl3;
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/gpapi/default.nix:26:0:
```
   |
26 |     description = "Google Play Unofficial Python API";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/gpapi/default.nix:27:5:
```
   |
27 |     maintainers = with maintainers; [ ];
   |     ^
```
</li>
  </ul>
</details>

**Review by fabaff — COMMENTED** [review]
The license seems `gpl3Only`. `nixpkgs-hammering` will stop complaining :wink:

I don't think that we can run the [tests](https://github.com/NoMore201/googleplay-api/tree/master/test) as token seems to be needed. Could you add `doCheck = false;` and `pythonImportsCheck` please.

File: pkgs/development/python-modules/gpapi/default.nix [hunk 1]
```diff
@@ -14,11 +18,15 @@ buildPythonPackage rec {
     sha256 = "0ampvsv97r3hy1cakif4kmyk1ynf3scbvh4fbk02x7xrxn4kl38w";
   };
 
-  propagatedBuildInputs = [ requests protobuf pycryptodome ];
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/python-modules/gpapi/default.nix:21]: please add a comment.

**Comment by risicle** [comment]
WFM nixos x86_64 & macos 10.15

Just needs the comment mentioned above...

**Comment by DavHau** [comment]
Added comment about disabled tests

**Pull request merged**